src/index.tsx

```diff
@@ -19,42 +19,36 @@ const IoLoginUtils = NativeModules.IoLoginUtils
 
 export type AreaError = 'NativeRedirectError' | 'NativeAuthSessionError';
 
-export type IosError =
-  | 'Invalid URL'
-  | 'Request Error'
-  | 'Invalid response'
-  | 'Redirecting Error'
-  | 'NativeAuthSessionClosed'
+export type IOSError =
+  | 'RequestError'
+  | 'InvalidResponse'
   | 'MissingResponseURL'
   | 'ErrorOnResponseOrNativeComponent'
   | 'GenericErrorOnResponse'
-  | 'NativeComponentNotInstantiated'
   | 'iOSVersionNotSupported'
-  | 'Redirecting Error-MissingURL';
+  | 'RedirectingErrorMissingURL'
+  | 'NativeAuthSessionClosed'
+  | 'NativeComponentNotInstantiated'
+  | 'RedirectingError';
 
 export type AndroidError =
-  | 'ErrorOnClientSync'
+  | 'MissingActivityOnPrepare'
+  | 'FirstRequestError'
+  | 'ConnectionRedirectError'
+  | 'BrowserNotFound'
   | 'NativeAuthSessionClosed'
   | 'NativeComponentNotInstantiated'
-  | 'First Request Error'
-  | 'Error while creating connection redirect'
-  | 'Redirecting Error'
-  | 'MissingDataFromIntent'
-  | 'CustomTabContextIsNull'
-  | 'CustomTabActivityContextIsNull'
-  | 'MissingBrowserPackageNameWhileOpening'
-  | 'MissingActivityOnInit'
-  | 'SyncTimeoutError'
-  | 'ErrorOnClientSync';
+  | 'RedirectingError'
+  | 'IllegalStateException';
 
-export type Error = IosError | AndroidError;
+export type Error = IOSError | AndroidError;
 
 export type LoginUtilsError = {
-  userInfo: {
-    Error: Error;
-    URL: string | undefined;
-    StatusCode: number | undefined;
-    Parameter: Array<string> | undefined;
+  userInfo?: {
+    error: Error;
+    url: string | undefined;
+    statusCode: number | undefined;
+    parameter: Array<string> | undefined;
```

Comment:
These are breaking changes, if we made those modifications we need to update the major version of the lib and describe the changes in the changelog of the new release. What do you think?

Reply:
Absolutely.
This change aligns the property names of `userInfo` (PascalCase) to the code style of the RN part (camelCase). Also, some errors may not have user info, so `userInfo` become nullable.